src/azure-cli/azure/cli/command_modules/profile/tests/latest/test_profile_custom.py

```diff
@@ -124,6 +124,24 @@ def test_login(msi_port, identity_id=None):
         # assert
         self.assertTrue(invoked)
 
+    @mock.patch('azure.cli.core._profile.SubscriptionFinder', autospec=True)
+    def test_login_invalid_url(self, mock_subscription_finder):
```

Comment:
Please make this a more general test case so that other errors can also be tested to avoid regression.